CSS/reset.css

```diff
@@ -0,0 +1,23 @@
+html {
+    box-sizing: border-box;
+    font-size: 16px;
+  }
+  
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  body, h1, h2, h3, h4, h5, h6, p, ol, ul {
+    margin: 0;
+    padding: 0;
+    font-weight: normal;
+  }
+  
+  ol, ul {
+    list-style: none;
+  }
+  
+  img {
+    max-width: 100%;
+    height: auto;
+  }
```

Comment:
Could you pls fix code formatting? Here we have only one problem - no indentations should be there before selectors - but it can deliver a lot of problems in more complicated code
Try to use "ctrl + shift + I" in VSC or open its keyboard settings (you need to google how to do that) and find hotkey for smth like "Format code" or "Autoformatting". If it will not work - you can google "как форматировать код в VS code"

Reply:
What do you mean? If you talk about lineheight, it was written so in the video tutorial. The value can be a number without naming.

Reply:
It happened because I copied this piece from the website and forgot to format it